profiles/default/linux/arm/17.0/musl/make.defaults

```diff
@@ -0,0 +1,4 @@
+# Copyright 1999-2014 Gentoo Foundation.
+# Distributed under the terms of the GNU General Public License v2
+
+CHOST="arm-unknown-linux-musleabi"
```

Comment:
You want to explicitly set CHOST_arm too.

Reply:
done